GitUI/CommandsDialogs/FormVerify.LostObject.cs

```diff
@@ -30,7 +30,7 @@ private sealed class LostObject
             private static readonly string LogCommandArgumentsFormat = (ArgumentString)new GitArgumentBuilder("log")
             {
                 "-n1",
-                "--pretty=format:\"%aN, %s, %ct, %P\" {0}"
+                "--pretty=format:\"%aN\u001F%s\u001F%ct\u001F%P\" {0}"
```

Comment:
May this sequence appear in the encoded text from Git?
(There were some discussions about that for the RevisionReader changes).
Regardless, this should work much better than  `, `.

Reply:
> May this sequence appear in the encoded text from Git?

Sorry, I don't understand. Can you elaborate more?

I try a probable answer...

When 'Unit Separator' 1F is displayed, it is invisible so the fields seems not be separated even if they are.
I admit it is not the easiest thing to debug even if it works well and this character has been created for that... ( "ASCII includes several control characters that are intended to be used as delimiters. They are: 28 for File Separator, 29 for Group Separator, 30 for Record Separator, and 31 for Unit Separator.") 

![image](https://user-images.githubusercontent.com/460196/150605073-63924724-c04c-4973-9f0e-a04078f03434.png)

Utf code used in VS:
![image](https://user-images.githubusercontent.com/460196/150605272-831102e3-5917-4cc4-929f-ad0dda15032b.png)

Reply:
The regex for the `subject` group is greedy including possible `\u001F`. Only the `author` group might be truncated by mistake. Then the right part would be added to the `subject`. So, I am pretty confident.

Reply:
> > May this sequence appear in the encoded text from Git?
> 
> Sorry, I don't understand. Can you elaborate more?
> When 'Unit Separator' 1F is displayed, it is invisible so the fields seems not be separated even if they are. I admit it is not the easiest thing to debug even if it works well and this character has been created for that... ( "ASCII includes several control characters that are intended to be used as delimiters. They are: 28 for File Separator, 29 for Group Separator, 30 for Record Separator, and 31 for Unit Separator.")

A string is not stored in utf8 in C#, so the inserted character could possibly appear in the encoded string and wreck the parsing. If the separator character is ensured to not appear in strings, this will be safe (and it is less of a problem than the current implementation regardless).